tasks/install.yaml

```
@@ -0,0 +1,30 @@
+- block:
+
+  - name: Add MariaDB repo key
+    apt_key:
+      url: https://mariadb.org/mariadb_release_signing_key.asc
+      state: present
+
+  - name: add mariadb repo
```

Inconsistent capitalization
```suggestion
  - name: Add MariaDB repository
```